### DISCUSSION TITLE
opencv version

### DISCUSSION THREAD

**Discussion by HyeonJungHam** [Nicholasli1995/EvoSkeleton#74] [Q&A] [answered]
Hello, I'm trying to run h36m2Dpose.py and found that the opencv is not included in the spec-list.txt (No module named 'cv2'). 
Could you please state what version of opencv is needed? 

Thank you, 
Hyeonjung Ham

**Comment by Nicholasli1995** [comment, marked as answer]
> Hello, I'm trying to run h36m2Dpose.py and found that the opencv is not included in the spec-list.txt (No module named 'cv2'). Could you please state what version of opencv is needed?
> 
> Thank you, Hyeonjung Ham

Hi, it shoube be opencv 3.4.2 according to my "conda list" command.

**Reply by HyeonJungHam** [reply]
Thank you for a quick response
